cpp/tests/experimental/spatial/point_in_polygon_test.cu

```diff
@@ -62,8 +62,8 @@ TYPED_TEST(PointInPolygonTest, OnePolygonOneRing)
   auto poly_point =
     this->make_device_points({{-1.0, -1.0}, {1.0, -1.0}, {1.0, 1.0}, {-1.0, 1.0}, {-1.0, -1.0}});
 
-  auto got      = rmm::device_vector<int32_t>(test_point.size());
-  auto expected = std::vector<int32_t>{false, false, false, false, true, true, true, true};
+  auto got      = rmm::device_vector<bool>(test_point.size() * poly_offsets.size());
+  auto expected = std::vector<bool>{false, false, false, false, true, true, true, true};
```

Comment:
`std::vector<bool>` *may be* a coalesced bit array (implementation defined). I'm not sure if `EXPECT_EQ` knows unpacking before comparing.

Reply:
https://github.com/rapidsai/cuspatial/issues/823

Reply:
Just change this to `thrust::host_vector<bool>` and it should work.